src/test/scala/nl.knaw.dans.easy.bagstore/ItemIdSpec.scala

```diff
@@ -18,10 +18,11 @@ package nl.knaw.dans.easy.bagstore
 import java.nio.file.Paths
 import java.util.UUID
 
-import scala.util.{ Failure, Success }
+
```

Comment:
```suggestion
```

Reply:
This looks like an extra newline, but I'm not entirely sure about this. Could you check and remove if necessary?